pantheon-sessions.php

```diff
@@ -470,7 +608,7 @@ public function primary_key_revert() {
 		// Remove table which did not function.
 		$query = "DROP TABLE {$temp_clone_table}";
 		$wpdb->query( $query );
-		$this->safe_output( __( 'Process complete.', 'wp-native-php-sessions' ), 'log' );
+		$this->safe_output( __( 'Blog processing complete.', 'wp-native-php-sessions' ), 'log' );
```

Comment:
More than other places, I think referring to sites as "blogs" here is notably anachronistic. It should be replaced with "Site".

Reply:
```suggestion
		$this->safe_output( __( 'Site processing complete.', 'wp-native-php-sessions' ), 'log' );
```